_opensearch/install/tar.md

```diff
@@ -7,144 +7,455 @@ nav_order: 50
+   # discovery.type to single-node or the bootstrap checks will
+   # fail when you try to start the service.
+   discovery.type: single-node
+
+   # If you previously disabled the security plugin in opensearch.yml,
+   # be sure to re-enable it. Otherwise you can skip this setting.
+   plugins.security.disabled: false
+   ```
+1. Save your changes and close the file.
+1. Specify an initial and max JVM heap size.
+   1.  Open `jvm.options`.
+         ```bash
+         vi /path/to/opensearch-{{site.opensearch_version}}/config/jvm.options
+         ```
+   1. Modify the values for initial and maximum heap size. As a starting point, you should set these values to half of the available system memory. For dedicated hosts this value can be increased based on your workflow requirements.
+      -  As an example, if the host machine has 8GB of memory then you might want to set the initial and maximum heap sizes to 4GB:
+         ```bash
+         -Xms4g
+         -Xmx4g
+         ```
+   1. Save your changes and close the file.
+1. Specify the location of the included JDK.
+   ```bash
+   export OPENSEARCH_JAVA_HOME=/path/to/opensearch-{{site.opensearch_version}}/jdk
    ```
-   # ======================== OpenSearch performance analyzer plugin config =========================
-
-   # NOTE: this is an example for Linux. Please modify the config accordingly if you are using it under other OS.
-
-   # WebService bind host; default to all interfaces
-   webservice-bind-host = 0.0.0.0
-
-   # Metrics data location
-   metrics-location = /dev/shm/performanceanalyzer/
-
-   # Metrics deletion interval (minutes) for metrics data.
-   # Interval should be between 1 to 60.
-   metrics-deletion-interval = 1
-
-   # If set to true, the system cleans up the files behind it. So at any point, we should expect only 2
-   # metrics-db-file-prefix-path files. If set to false, no files are cleaned up. This can be useful, if you are archiving
-   # the files and wouldn't like for them to be cleaned up.
-   cleanup-metrics-db-files = true
 
-   # WebService exposed by App's port
-   webservice-listener-port = 9600
+### Configure TLS
 
-   # Metric DB File Prefix Path location
-   metrics-db-file-prefix-path = /tmp/metricsdb_
+TLS certificates provide additional security for your cluster by allowing clients to confirm the identity of hosts and encrypt traffic between the client and host. For more complete information, refer to [Configure TLS Certificates]({{site.url}}{{site.baseurl}}/security-plugin/configuration/tls/) and [Generate Certificates]({{site.url}}{{site.baseurl}}/security-plugin/configuration/generate-certificates/) which are covered in the [Security Plugin]({{site.url}}{{site.baseurl}}/security-plugin/index/) documentation. For work performed in a development environment, self-signed certificates are usually adequate. This section will guide you through the basic steps required to generate your own TLS certificates and apply them to your OpenSearch host.
```

Comment:
Insert comma after the closing parenthesis "...-certificates/)"

Reply:
Done thank you!